tls/s2n_cipher_preferences.c

```diff
@@ -794,8 +913,10 @@ const struct s2n_cipher_preferences cipher_preferences_cloudfront_upstream_tls12
 };
 
 struct s2n_cipher_suite *cipher_suites_cloudfront_tls_1_2_2019[] = {
-    &s2n_ecdhe_rsa_with_aes_256_gcm_sha384,
+    S2N_TLS13_CLOUDFRONT_CIPHER_SUITES_20200716,
```

Comment:
Rather weird that a `tls_1_2`  policy contains `TLS13_CIPHERS` - but I guess that is because the first name indicates the minimum version whereas the others is actual versions?

Reply:
yes, tls_1_2 refers to minimum version not maximum version